gbdk-lib/examples/gb/template_minimal/Makefile

```diff
@@ -22,7 +22,7 @@ all:	$(BINS)
 
 compile.bat: Makefile
 	@echo "REM Automatically generated from Makefile" > compile.bat
-	@make -sn | sed y/\\//\\\\/ | grep -v make >> compile.bat
+	@make -sn | sed y/\\//\\\\/ | sed/mkdir -p/mkdir/ | grep -v make >> compile.bat
```

Comment:
The added invocation of sed above is not going to work (at least on Linux) for two reasons:
*  `sed` needs a space after it in order to be invoked properly
* The sed pattern match doesn't follow the syntax 

Probably what you want instead is: 
`	@make -sn | sed y/\\//\\\\/ | sed s/mkdir\ \-p/mkdir/ | grep -v make >> compile.bat`